src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "core.basesyntax.Ball{"
+                + "color='" + color
+                + '\''
+                + ", number=" + number
+                + '}';
+    }
+}
```

Comment:
It's a good practice to include access modifiers for the class constructor and its methods. Consider adding an access modifier to the constructor and the toString() method to explicitly define their visibility.